fdbbackup/backup.actor.cpp

```diff
@@ -2692,6 +2697,24 @@ static void reportBackupQueryError(UID operationId, JsonBuilderObject& result, s
 	TraceEvent("BackupQueryFailure").detail("OperationId", operationId).detail("Reason", errorMessage);
 }
 
+std::pair<Version, Version> getMaxMinRestorableVersions(const BackupDescription& desc, bool mayOnlyApplyMutationLog) {
```

Comment:
I don't understand why `mayOnlyApplyMutationLog` is `!keyRangesFilter.empty()`.

Reply:
I don't understand that either. The code simply preserves previous behavior. Looks like if backup doesn't have a restorable version, and it has KeyRangeFilter set, it just tries to use the mutation log only.

I don't see this logis is used anywhere else. I can also remove it.

Reply:
I'm fine either way. If we keep this logic, maybe add a comment on this.

Reply:
I added a comment. This option (hopefully) won't be used in our case.